src/Kunstmaan/NodeBundle/Resources/config/services.yml

```diff
@@ -120,3 +120,21 @@ services:
         class: Kunstmaan\NodeBundle\Command\FixTimestampsCommand
         calls:
             - [setContainer, ["@service_container"] ]
+
+    kunstmaan_node.slug.listener:
+        class: Kunstmaan\NodeBundle\EventListener\SlugListener
```

Comment:
This class should be a parameter, so users can easily override it with their own implementation.